procedures/foreman/branch.md.erb

```diff
@@ -1,5 +1,11 @@
 [ ] Make this post a wiki
 
+## Roles
+
+* Release Owner: <%= owner %>
+* Release Engineer: <%= engineer %>
+* Installer Maintainer: @
```

Comment:
Is this intentionally left empty?

Reply:
Yes, it was sort of a one-off and I wasn't entirely sure if we should add it to the branch command.